src/experiments.js

```diff
@@ -47,6 +51,102 @@ export function isCanary(win) {
 }
 
 
+function getExperimentsKey_() {
+  let key;
+  const url = '' //TODO(kmh287): fill this in
+  if (url) {
+    return xhrFor(this.win).fetchJson(url, {
+      mode: 'cors',
+      method: 'GET',
+      ampCors: false,
+      credentials: 'omit',
+    }).then(jwkObj => {
```

Comment:
How can this be tested? I'm not sure how to generate a JSON web key nor how to use it to sign some fake data.